_config.yml

```diff
@@ -15,3 +15,7 @@ exclude:
     - 'vendor'
 sass:
     style: compressed
+languages: ["cn", "de", "en", "ru", ]
+default_lang: "en"
```

Comment:
We need `https://tc39.es/en/`, no? Not familiar w/ this plugin yet, but the `permalink` attribute in the front matter of the English version should be very useful for this.

Reply:
I am waiting on some unanswered questions before doing this. It can be covered in a follow up.